src/main.py

```diff
@@ -16,15 +17,29 @@
 )
 from utils.conversions import rpm_to_rad_s, deg_to_rad
 
+# Define parser
+parser = argparse.ArgumentParser(description="Simulate a Baja SAE car")
 
+# Define arguments
+parser.add_argument(
+    "--incline_angle",
+    type=float,
+    default=0.0,
+    help="Incline angle in degrees (default: 0.0)"
+)
+
+# Parse arguments
+args = parser.parse_args()
```

Comment:
This part should def happen in a different file, otherwise it will get majorly out of control when we have 20+

Reply:
added